packages/manager/apps/pci-instances/src/pages/instances/Instances.page.tsx

```diff
@@ -0,0 +1,300 @@
+      addFilter({
+        key: 'search',
+        value,
+        comparator: FilterComparator.Includes,
+        label: 'name',
+      });
+    },
+    [addFilter],
+  );
+
+  if (data && !data.length && !filters.length && !isFetching)
+    return <Navigate to={SECTIONS.onboarding} />;
+
+  if (routeLoaderData?.notFoundAction) {
+    return <NotFoundPage />;
+  }
+
+  return (
+    <>
+      <PageLayout>
+        {project && <Breadcrumb projectLabel={project.description ?? ''} />}
+        <div className="header mb-6 mt-8">
+          <div className="flex items-center justify-between">
+            <Title>{t('common:pci_instances_common_instances_title')}</Title>
+            <div className="flex items-center flex-wrap">
+              <ChangelogButton links={CHANGELOG_LINKS} />
+              <PciGuidesHeader category="instances" />
+            </div>
+          </div>
+        </div>
+        <div>
+          <OsdsDivider />
+          <Notifications />
+          <SearchNotifications />
+          <OsdsDivider />
+          <div className="sm:flex items-center justify-between mt-4 min-h-[40px]">
+            <OsdsButton
+              size={ODS_BUTTON_SIZE.sm}
+              variant={ODS_BUTTON_VARIANT.stroked}
+              color={ODS_THEME_COLOR_INTENT.primary}
+              inline
+              href={createInstanceHref}
+            >
+              <span slot="start" className="flex items-center">
+                <OsdsIcon
+                  name={ODS_ICON_NAME.ADD}
+                  size={ODS_ICON_SIZE.xxs}
+                  color={ODS_THEME_COLOR_INTENT.primary}
+                  className="mr-4"
+                />
+                <span>{t('common:pci_instances_common_create_instance')}</span>
+              </span>
+            </OsdsButton>
+            <div className="justify-between flex gap-5 min-h-[40px] items-center">
+              <div className="flex items-center">
+                {isRefetching ? (
+                  <OsdsSpinner inline={true} size={ODS_SPINNER_SIZE.md} />
+                ) : (
+                  <OsdsButton
+                    slot="reset-trigger"
+                    size={ODS_BUTTON_SIZE.sm}
+                    color={ODS_THEME_COLOR_INTENT.primary}
+                    variant={ODS_BUTTON_VARIANT.stroked}
+                    onClick={handleRefresh}
+                    {...(isFetching && { disabled: true })}
+                  >
+                    <span slot="start" className="flex items-center mr-0">
+                      <OsdsIcon
+                        name={ODS_ICON_NAME.REFRESH}
+                        size={ODS_ICON_SIZE.sm}
+                        color={ODS_THEME_COLOR_INTENT.primary}
+                      />
+                    </span>
+                  </OsdsButton>
+                )}
+              </div>
+              <SearchBar
+                onSearchSubmit={handleSearchSubmit}
+                isDisabled={filters.length > 0 || isFetching}
```

Comment:
This seems weird, you can search and then apply a filter but not the other way around ?

Reply:
I'm not sure to understand this comment, we want to disable the search bar when a filter is already applied, as API does not support multiple filters at the same time, and also when a query is fetching.
We can remove this condition but it means that if a filter is already applied, it will be automatically replaced bu what have been re-entered in the search bar